lib/package-manager.coffee

```
@@ -230,7 +231,7 @@ class PackageManager
     apmProcess = @runCommand(args, exit)
     handleProcessErrors(apmProcess, errorMessage, onError)
 
-  unload: (packageName) ->
+  unload: (name) ->
```

I think the promise catch was eating any errors too.